Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -62,7 +62,7 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging {
      - parameter desiredSimulationMode: The desired simulation mode to use.
      - returns: A navigation service that manages location updates along `route`.
      */
-    func carPlayManager(_ carPlayManager: CarPlayManager, navigationServiceAlong route: Route, routeIndex: Int, routeOptions: RouteOptions, desiredSimulationMode: SimulationMode) -> NavigationService
+    func carPlayManager(_ carPlayManager: CarPlayManager, navigationServiceAlong routeResponse: RouteResponse, routeIndex: Int, routeOptions: RouteOptions, desiredSimulationMode: SimulationMode) -> NavigationService
```

Comment:
Documentation is outdated.

Reply:
I have a strange feeling reading `navigationServiceAlong routeResponse`. How is that possible to navigate along response? It might be a consequence of the wrong abstraction or a better naming can be found.